src/base/utils/string.cpp

```diff
@@ -133,14 +133,24 @@ bool NaturalCompare::lessThan(const QString &left, const QString &right) const
 bool Utils::String::naturalCompareCaseSensitive(const QString &left, const QString &right)
 {
     // provide a single `NaturalCompare` instance for easy use
-    static NaturalCompare nCmp(true);  // this is thread-safe in C++11 (stated in spec 6.7.4)
+    // https://doc.qt.io/qt-5/threads-reentrancy.html
+#ifdef Q_OS_OSX  // workaround for TravisCI
```

Comment:
Only for travis-ci? I suppose regular OSX builds will need this, right? If so please remove.
And also squash this commit into previous one.